RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ---
 * [*] [a11y] Improve text read by screen readers for BottomSheetSelectControl [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4854]
+* [*] [Image block] Add Insert from URL as a media option
```

Comment:
We usually add a reference to the Gutenberg Mobile PR along with the entry, e.g.:
```
[a11y] Improve text read by screen readers for BottomSheetSelectControl [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4854]
```
Could you also add it for this change? Thanks 🙇 !